src/steamship/agents/schema/tool.py

```diff
@@ -40,6 +40,12 @@ class Tool(BaseModel):
     """Human-friendly description.
     Used for logging, tool indices, etc."""
 
+    return_direct: bool = False
+    """Whether to return the tool's output directly.
+
+    Setting this to True means that after the tool is called, the AgentExecutor will stop looping.
```

Comment:
```suggestion
    Setting this to True means that after the Tool is run, the Tool's output will be returned without further Agent interactions. This will preempt the execution of other tools, as well as any subsequent calls to LLMs, etc., to interpret the Tool output.
```